tests/test_routes_store.py

```
@@ -154,3 +154,63 @@ async def test_uninstall_not_installed_fails(self, store_client):
+        ):
+            instance = MockInstaller.return_value
+            instance.install = AsyncMock(return_value=mock_result)
+
+            resp = await store_client.post("/api/store/install-v2", json={
+                "app_id": "gitea-lxc",
+                "admin_password": "secret",
+            })
+
+        assert resp.status_code == 200
+        assert len(update_calls) == 1
+        app_id, host, port, backend, ui_path = update_calls[0]
```

Ruff (0.15.11)

[warning] Unpacked variable `ui_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)